SMT-LIB/parse_all.py

```diff
@@ -53,27 +54,39 @@ def main():
     parser.add_argument('--out', type=str, default="stats.out", nargs='?',
                         help='Where to save the statistics')
 
+    parser.add_argument('--reps', type=int, default=1,
+                        help="number of times each benchmark is run")
+
     args = parser.parse_args()
 
     random.seed(42)
-    p = multiprocessing.Pool()
     chunks = multiprocessing.cpu_count()
     file_list = list(get_all_smt_files(args.base))
     random.shuffle(file_list)
     if args.count == -1:
         files_cnt = len(file_list)
     else:
         files_cnt = args.count
-    print("Submitting %d jobs, %d at the time" % (files_cnt, chunks))
-    timings = p.map(execute_script_fname, islice(file_list, files_cnt), chunks)
+
+    timings = []
+    errors = []
+
+    with multiprocessing.Pool() as p:
+        print(f"Submitting {files_cnt} jobs, {chunks} at the time")
+        for el in p.starmap(execute_script_fname,
```

Comment:
Minor: Wouldn't `repeat(zip(islice(...)), args.reps)` achieve the same and avoid requiring passing `reps` to `execute_script_fname`.
It seems this is the only reason you are extending `execute_script_fname` with reps.